mozci/console/commands/push.py

```diff
@@ -916,14 +925,16 @@ def handle(self) -> None:
                                 "type": "comment",
                             }
                         )
+                        if config.get("warnings", {}).get("inconsistent", False):
+                            warnings_to_notify.add(warnings[-1]["message"])
 
             # Output all warnings and also send them to the Matrix room if defined
             matrix_room = config.get("matrix-room-id")
             for warning in warnings:
                 warn_type = warning["type"]
                 warn_message = warning["message"]
                 self.line(f"<{warn_type}>{warn_message}</{warn_type}>")
-                if matrix_room:
+                if matrix_room and warn_message in warnings_to_notify:
```

Comment:
Instead of using a new set maybe we could just add a new boolean attribute `to_notify` in each warning dict from the `warnings` list, WDYT?

Reply:
Good idea, done in ed9738f.